### PR TITLE
Mat 4629 population basis enter issue

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4629](https://jira.cms.gov/browse/MAT-4629)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [x] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureGroups/MeasureGroups.tsx [hunk 1]
```diff
@@ -504,6 +504,11 @@ const MeasureGroups = () => {
                         renderInput={(params) => (
                           <TextField {...params} label="" />
                         )}
+                        onKeyDown={(e) => {
+                          if (e.keyCode === 13) {
+                            e.preventDefault();
```
Comment (nmorasb) [inline src/components/measureGroups/MeasureGroups.tsx]: seems like there have a been a number of these now. Is it maybe time to create a util function to do this? i.e.
`onKeyDown={preventSubmitOnEnter}`
```
export function preventSubmitOnEnter(e) {
  if (e.keyCode === 13) {
    e.preventDefault()
  }
}
```

File: src/components/measureGroups/MeasureGroups.tsx [hunk 2]
```diff
@@ -504,6 +504,11 @@ const MeasureGroups = () => {
                         renderInput={(params) => (
                           <TextField {...params} label="" />
                         )}
+                        onKeyDown={(e) => {
```
Comment (adongare) [inline src/components/measureGroups/MeasureGroups.tsx]: this can be handled in a different way. Looks like whenever a new field is introduced, we need this code. I looked into it lil bit and seems like event is getting propaugated to `+ Add Measure Group` link(actually a button which I think should be a link). Having `type="button"` attribute on `add measure group` button would fix this or even you can replace button element with with `DSLink` from the design system.

Comment (sb-cecilialiu) [reply src/components/measureGroups/MeasureGroups.tsx]: @adongare @nmorasb Thanks so much Nate and Ashok! I will change the button to DSLink and revert some some changes for the fix before.

**Review by adongare — APPROVED** [review]

**Review by nmorasb — APPROVED** [review]

File: src/components/measureGroups/MeasureGroups.tsx (hunk 1 above)
Comment (sb-cecilialiu) [reply src/components/measureGroups/MeasureGroups.tsx]: Nate had a look again and he is ok with the change of switching to DSLink.
